### PR TITLE
ISSUE-32 add google analytics opt out option

### PR DESCRIPTION
**Pull request by tabanliviu**
implements #32 
- add better description and options page layout - DONE
@simonhearne / @andydavies can you please look at the branch and suggest some descriptive text about the google analytics option

### CODE REVIEW

**Review by simonhearne — COMMENTED** [review]
I've added some text to describe what the checkbox does. Do we need to include a full privacy policy?

**Comment by tabanliviu** [comment]
@simonhearne if you think it helps? perhaps a link to a predefined one?

File: app/options.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+<!DOCTYPE html>
+<html>
+<head>
+  <title>NCC Image Checker - Options</title>
+  <link rel="stylesheet" href="styles/options.css">
+  <script src="analytics.js"></script>
+</head>
+<body>
+  <div class="text">
+    <p>NCC Group collects information about your use of this extension, including how many times it is opened and what pages it is used on, using Google Analytics.</p>
```
Comment (tabanliviu) [inline app/options.html]: @simonhearne MAINTENANCE - move the description paragraphs inside the options div so it's colocated with the feature it's describing

**Comment by Ian-Hi** [comment]
Verified ok

File: app/analytics.js [hunk 2]
```diff
@@ -14,11 +14,22 @@
...
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-97952592-1', 'auto');
+  // Removes failing protocol check. @see: http://stackoverflow.com/a/22152353/1958200
+  ga('set', 'checkProtocolTask', null);
+
+  function initGa() {
```
Comment (MalcolmSmithUK) [inline app/analytics.js]: @tabanliviu CONSISTENCY - Function call adds more barriers than it assists with.

**Review by MalcolmSmithUK — CHANGES_REQUESTED** [review]

File: app/analytics.js (hunk 2 above)
Comment (tabanliviu) [reply app/analytics.js]: OK

**Comment by tabanliviu** [comment]
@MalcolmSmithUK ready for review

**Review by MalcolmSmithUK — APPROVED** [review]

File: app/options.html (hunk 1 above)
Comment (MalcolmSmithUK) [reply app/options.html]: @tabanliviu This appears to still be a thing.

**Pull request merged**